Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -952,6 +955,25 @@ void UpdateBarTextColor()
 				_tabLayout.TabTextColors = colors;
 		}
 
+		protected virtual void UpdateBarSelectedTextColor()
+		{
+			AColor selectedColor;
+			AColor unselectedColor;
+
+			if (Element != null)
+			{
+				selectedColor = (Element.BarSelectedTextColor.IsDefault ? Element.SelectedTabColor : Element.BarSelectedTextColor).ToAndroid();
```

Comment:
I feel like this is going to make the text color the same as the tab background color, and thus not visible, if the text color isn't set. Is this really what it was doing before?

Reply:
I had to re-read this code a few times to remember what I was doing and hook up the debugger. It isn't super clear and maybe we can come up with better variable names as it is one of those cases where every part of the variable name is super important

This code will use the selected/unselected Tab Color if there is no value set for the Text Color. The Tab Color is not the same as the background color but the color of the icon to make them match. If the selected text color or unselected text color is set it will use those values.

The idea here is to sync up the foreground color of the text and icon in the tab. If the code doesn't explicitly set the color it will match the icon. This is where `SelectedTabColor` and `UnselectedTabColor` come into play.

Did I clear things up or did I just make it more confusing 😕